rust/release/src/versions_file.rs

```diff
@@ -205,7 +205,11 @@ impl VersionsFile {
             let product = current.release.product;
             let repo = Repository::open_from_env()?;
             let ts = repo.find_commit(current.commit)?.time().seconds();
-            let time = chrono::Utc.timestamp(ts, 0).to_rfc3339_opts(SecondsFormat::Secs, true);
+            let time = chrono::Utc
+                .timestamp_opt(ts, 0)
+                .single()
+                .expect("a single timestamp")
```

Comment:
my understanding is that this string should give the reason, not the requirement (i.e. “git timestamps are valid”)

Reply:
Your suggestion is definitely better, but it's also true that it expected a single timestamp 😅